tests-example/tilt_modules/extensions.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
should we .gitignore tilt_modules? i think we do in most cases...

Reply:
i thought it was [best practice](https://docs.tilt.dev/extensions.html#commit-to-source-control) to put your extensions in version control? (of course that may be for real projects and not example repos -- happy to roll with whatever you think is best.)

Reply:
hmmm...i'm not sure who wrote that best practice, but...it's not like tilt_modules invented vendoring, there's tons and tons of articles about the decision tree of when to vendor.

For this use-case, I would expect that we want to always use the latest version, since that's what people will be getting when they start their own projects.